src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (numberOne) => {
+    const result = (numbertwo) => {
+      const sum = numberOne + numbertwo;
```

Comment:
Just to simplify, you can directly return the result of this expression.
```suggestion
      return numberOne + numbertwo;
```